chains/lisk/dispatcher.go

```diff
@@ -1,27 +1,38 @@
 package lisk
 
 import (
+	"encoding/hex"
+
 	"github.com/sisu-network/deyes/chains"
 	"github.com/sisu-network/deyes/types"
+	"github.com/sisu-network/lib/log"
 )
 
 type LiskDispatcher struct {
+	chain  string
 	client LiskClient
 }
 
-func NewDispatcher(client LiskClient) chains.Dispatcher {
+func NewDispatcher(chain string, client LiskClient) chains.Dispatcher {
 	return &LiskDispatcher{
 		client: client,
+		chain:  chain,
 	}
 }
 
 func (d *LiskDispatcher) Start() {
+
 }
 
 func (d *LiskDispatcher) Dispatch(request *types.DispatchedTxRequest) *types.DispatchedTxResult {
+	trans, err := d.client.CreateTransaction(hex.EncodeToString(request.Tx))
+	if err != nil {
+		log.Errorf("Failed to create transaction, err = %v", err)
+		return nil
```

Comment:
Don't return nil. Return `DispatchedTxResult` with success = false and the error